app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
```

Comment:
Can you rewrite this for efficiency? Hint: in dictionary finding an element by a key has the best possible algorythmic coplexity